src/main.rs

```diff
@@ -215,7 +215,15 @@ async fn main() -> Result<()> {
             }
             let token =
                 AuthToken::from_str(&token).context("Wrong format for authentication token")?;
-            get_interactive(*hash.as_hash(), opts, token, out).await?;
+            tokio::select! {
```

Comment:
This is an oddity of the `tokio::select!` API, but it is almost always better to use the biased form since it avoids generating a random number on each call (not that this is in a loop so it doesn't matter much).  It also makes it easier to reason about which branch will be preferred.  For example here I'd prefer the get_interactive branch first if both ctrl-c and get_interactive complete at the same time.  That is the opposite order than they are written now.
```suggestion
            tokio::select! {
                biased;
```

Reply:
ahhh yes, I learned this while reviewing one of your PRs & I forgot to incorporate it. Thank you.